src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IArgument.vb

```diff
@@ -835,5 +758,110 @@ BC30272: 'y' is not a parameter of 'Public Sub M2(x As Integer)'.
             VerifyOperationTreeAndDiagnosticsForTest(Of InvocationExpressionSyntax)(source, expectedOperationTree, expectedDiagnostics)
         End Sub
 
+
+
+        <CompilerTrait(CompilerFeature.IOperation)>
```

Comment:
><CompilerTrait(CompilerFeature.IOperation)> [](start = 8, length = 43)

It looks like there are three new lines between this test and the one above. We usually use one line to separate tests.